pkgs/development/python-modules/labelbox/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, affine
+, attrs
+, certifi
+, chardet
+, click-plugins
+, click
+, cligj
+, idna
+, jinja2
+, markupsafe
+, numpy
+, pillow
+, pyparsing
+, rasterio
+, requests
```

Comment:
i would remove all the inputs you dont need, in this case you should just need `lib, buildPythonPackage, fetchPypi, requests`

Reply:
Done.